sky/cli.py

```diff
@@ -2213,35 +2213,41 @@ def _hint_for_down_spot_controller(controller_name: str):
            f'aware of the following:{colorama.Style.RESET_ALL}'
            '\n * All logs and status information of the spot '
            'jobs (output of `sky spot queue`) will be lost.')
+    if cluster_status == global_user_state.ClusterStatus.INIT:
+        msg += ('\n * If there are pending/in-progress spot jobs, those '
+                'resources will not be terminated and require manual cleanup.')
+    click.echo(msg)
     if cluster_status == global_user_state.ClusterStatus.UP:
-        try:
-            spot_jobs = core.spot_queue(refresh=False)
-        except exceptions.ClusterNotUpError:
-            # The spot controller cluster status changed during querying
-            # the spot jobs, use the latest cluster status, so that the
-            # message for INIT and STOPPED states will be correctly
-            # added to the message.
-            cluster_status = backend_utils.refresh_cluster_status_handle(
-                controller_name)
-            spot_jobs = []
+        with backend_utils.safe_console_status(
+                '[bold cyan]Checking spot jobs on the controller[/]'):
```

Comment:
UX: Currently I see this spinner spinning for a few seconds, then it disappears and we show `Proceed? [y/N]:`. Would it be possible to also print e.g., "No in-progress jobs found. It should be safe to terminate, but see the warning above." This will orient the user a bit better, I think.

Reply:
Good point! Added. Thanks!